Controller/ExceptionController.php

```diff
@@ -89,11 +89,11 @@ protected function createView(\Exception $exception, $code, Request $request, $s
     /**
      * Determines the status code to use for the response.
      *
-     * @param \Exception $exception
+     * @param \Throwable $exception
      *
      * @return int
      */
-    protected function getStatusCode(\Exception $exception)
+    protected function getStatusCode(\Throwable $exception)
```

Comment:
I am afraid that this is a BC break for child classes that we cannot make: https://3v4l.org/cLLKB

A possible solution could be to add a `private` `getStatusCodeFromThrowable()` method that will be called when the `$exception` argument in `showAction()` does not extend the `\Exception` class.